cmd/logs.go

```diff
@@ -26,106 +28,149 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	maxEsLogsPerQuery     = 10000
+	fourteenDaysInSeconds = 60 * 60 * 24 * 14
+)
+
 // numberOfLines of logfile to return
 var numberOfLines = "400"
```

Comment:
@DockToFuture , @KristianZH what is the purpose of this? Do we still need it as now we have a `--tail` param?